src/goto-instrument/module_dependencies.txt

```diff
@@ -5,6 +5,7 @@ assembler
 cpp
 goto-instrument
 goto-programs
+json
```

Comment:
Needed?

Reply:
Yes, this is needed.

Reply:
Yes, we get cpplint errors without this.

Reply:
Interesting...  Thanks.